app/components/Form/ProjectManagerForm.tsx

```diff
@@ -135,13 +135,43 @@ const ProjectManagerForm: React.FC<Props> = (props) => {
   const [applyUpdateFormChangeMutation] = useUpdateFormChange();
 
   // Delete a manager from the project revision
-  const [discardFormChange, discardInFlight] = useDiscardFormChange();
+  const [deleteManager, deleteManagerInFlight] =
+    useDeleteManagerFromRevisionMutation(change.formChange?.operation);
   const handleClear = () => {
-    discardFormChange({
-      formChange: {
-        id: change.formChange.id,
-        operation: change.formChange.operation,
-      },
+    let variables;
+    let optimisticResponse;
+    // If the form_change has a CREATE operation then we want to delete the form_change from the database
+    if (change.formChange?.operation === "CREATE")
+      variables = {
+        input: {
+          id: change.formChange.id,
+        },
+        projectRevision: projectRevisionId,
+      };
+    // If the form_change has an UPDATE operation then we want to update the form_change in the database with operation = 'ARCHIVE'
+    else {
+      variables = {
+        input: {
+          id: change.formChange.id,
+          formChangePatch: {
+            operation: "ARCHIVE",
+          },
+        },
+        projectRevision: projectRevisionId,
+      };
+      optimisticResponse = {
+        updateFormChange: {
+          formChange: {
+            id: change.formChange.id,
+            operation: "ARCHIVE",
+            newFormData: {},
+          },
+        },
+      };
+    }
+    deleteManager({
+      variables,
+      optimisticResponse,
```

Comment:
Can we update `useDiscardFormChange` instead to have an archive and delete mutation as an option, so that the branching logic on `operation === "CREATE"` isn't duplicated?